pkg/generate/git/repository.go

```diff
@@ -465,3 +465,19 @@ func IsExitCode(err error, exitCode int) bool {
 	}
 	return false
 }
+
+func gitBinary() string {
+	if runtime.GOOS == "windows" {
+		return "git.exe"
+	}
+	return "git"
+}
+
+func IsGitInstalled() bool {
```

Comment:
is timedCommand ever actually used for any command other than "git"?  since timedCommand already includes git-specific logic (the gitError it returns), it seems like we might as well go the rest of the way, in which case we don't need this.

Alternatively/in-addition "isGitBinaryInstalled" could/should be renamed to "isBinaryInstalled" since it's a generic check.